### DISCUSSION TITLE
Audio file display format

### DISCUSSION THREAD

**Discussion by heerbovec** [mediacms-io/mediacms#276] [Q&A] [answered]
I am planning to use MediaCMS as an audio archive. If I add a poster picture, the width of the player significantly outspaces the picture on both sides. Can a solution be configured?

Best,
h.h.

**Comment by heerbovec** [comment]
I found three files containing *video* formatting definitions (560x315px) :

src/static/js/utils/settings/optionsEmbedded.js
config/installation/features.html
src/templates/config/installation/features.config.js

None of them seem to directly change the audio player's output format. What am I missing?

Best regards,
h.h.

**Comment by mgogoulos** [comment]
Hi @heerbovec , I totally missed your previous message, apologies. 

@styiannis , we need your help on this!

**Comment by heerbovec** [comment]
Thank you for that.

By the way, this is the site I want to transfer to mediacms: http://audiothek.philo.at. It is the largest german-language audio archive for philosophy (running on seriously outdated software).

**Comment by mgogoulos** [comment, marked as answer]
I now see what you mean. This is how poster appears on an audio file
![css_contain](https://user-images.githubusercontent.com/1027116/130326095-27f0a5fe-414a-4bbf-a2bb-01a5906ed351.png)

However, if you want to get the poster to get more area, and this is better for you:

![css_cover](https://user-images.githubusercontent.com/1027116/130326280-f7043887-1399-412a-bda1-c5d69d54dc11.png)

Then there is a quick solution: 

1. edit file static/css/_extra.css , basically append the following css rule
```
.video-js.vjs-mediacms.vjs-audio .vjs-poster {
    background-size: cover;
}
```
2. restart MediaCMS, if you're using the Single Server installation, 
```
root@server /home/mediacms.io/mediacms # systemctl restart mediacms
```

while if you're using Docker installation:
```
root@server /root/mediacms # docker-compose restart web
```

This can work as a quick fix, let us know if this is better. We can consider adding as the default behavior (let's wait for @styiannis 's feedback since he has done the work there, to tell us if there's a better way to do it)


I'd love to help you migrate content for https://audiothek.philo.at , keep us posted here or connect through info@mediacms.io

We've mostly focused on video + image support, but definitely are interested to optimize audio experience, and also fix any issues that might exist. 

Best
Markos

**Reply by heerbovec** [reply]
Thanks. This works, but at a price. One has to be careful to pick appropriate pictures and the width of the player is unfamiliar for audio files. It would be nice if one could somehow shrink it :-)

Best,
h.h.
